main:
  hz                      : 50 
  spinner_count           : 3
  driving_mode_changeable : 1         ## 주행모드 변경 가능 여부 (0:no, 그외:ok)
  joystick_control        : 1         ## joystick 제어 가능 여부 (0:no, 그외:ok)
  propulsion_scale_factor : -4.8       ## joystick scale factor
  steer_scale_factor      : 100.0       ## joystick scale factor

alive:
  hz                  : 20            ## heartbeat 보고 주기

logger:
  debug               : 1             ## debug 옵션 (0: off, 그외 : on)
  hz                  : 50            ## thread 동작 속도
  suffix              : _trashbot     ## 로그 파일 이름 뒤에 붙는 suffix
  dir_name            : log           ## 로그 파일 저장 디렉토리
  max_log_per_write   : 50            ## 한번에 기록 할 최대 로그 갯수
  auto_write_interval : 500           ## millisecond 로그 파일을 자동으로 기록하는 주기